components/badge/stories/template.js

```diff
@@ -52,3 +53,13 @@ export const Template = ({
 		</div>
 	`;
 };
+
+/* Displays icon-only, text-only, and icon-and-text badge options. */
+export const ContentOptions = (args, context) => Container({
+	withBorder: false,
+	content: html`
+		${Template(args, context)}
+		${Template({ ...args, iconName: undefined }, context)}
+		${Template({ ...args, label: undefined }, context)}
+	`
```

Comment:
I mentioned this in the other PR, this is an alternate way to define the container content if you don't want to have to import the html literal here.

```suggestion
	content: [
		Template(args, context).
		Template({ ...args, iconName: undefined }, context),
		Template({ ...args, label: undefined }, context),
	]
```